index.html

```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
           <li>Terminal</li>
         </ul>
       </section>
+      <section class="grid work">
+        <div class="container handles">
+          <img src="media/icons/github.svg" alt="github">
+          <img src="media/icons/linkedin.svg" alt="linkedin">
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
+        <div>
+          <h4>↓ Work</h4>
+        </div>
+        <div>
+          <h2>MY RECENT <br> WORK</h2>
+        </div>
```

Comment:
- [ ] I think this heading of MY RECENT WORK, should be in the next section of your projects. It's a heading that describes what that section is all about.
- [ ] Also there is an overlap of this heading with the first image of your projects. The heading is completely hidden behind the image. I had to increase the screen width to notice that it was hidden behind. See screenshot below:
![image](https://user-images.githubusercontent.com/60772732/125181722-aa991000-e1ff-11eb-91e3-3ce8dbbba518.png)

Reply:
Well resolved